[2024-12-17T10:54:11.121+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-17T10:54:11.129+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: call_9863800249_2024-12-17T10-52-04-719125Z.check_if_callerid_is_present_in_calllogs_table scheduled__2024-12-17T10:52:04.719125+00:00 [queued]>
[2024-12-17T10:54:11.132+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: call_9863800249_2024-12-17T10-52-04-719125Z.check_if_callerid_is_present_in_calllogs_table scheduled__2024-12-17T10:52:04.719125+00:00 [queued]>
[2024-12-17T10:54:11.132+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-17T10:54:11.138+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): check_if_callerid_is_present_in_calllogs_table> on 2024-12-17 10:52:04.719125+00:00
[2024-12-17T10:54:11.142+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=140132) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-17T10:54:11.143+0000] {standard_task_runner.py:72} INFO - Started process 140133 to run task
[2024-12-17T10:54:11.143+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'call_9863800249_2024-12-17T10-52-04-719125Z', 'check_if_callerid_is_present_in_calllogs_table', 'scheduled__2024-12-17T10:52:04.719125+00:00', '--job-id', '667', '--raw', '--subdir', 'DAGS_FOLDER/complete_dag.py', '--cfg-path', '/tmp/tmptdfzqvyd']
[2024-12-17T10:54:11.144+0000] {standard_task_runner.py:105} INFO - Job 667: Subtask check_if_callerid_is_present_in_calllogs_table
[2024-12-17T10:54:11.165+0000] {task_command.py:467} INFO - Running <TaskInstance: call_9863800249_2024-12-17T10-52-04-719125Z.check_if_callerid_is_present_in_calllogs_table scheduled__2024-12-17T10:52:04.719125+00:00 [running]> on host 1868de310944
[2024-12-17T10:54:11.201+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='call_9863800249_2024-12-17T10-52-04-719125Z' AIRFLOW_CTX_TASK_ID='check_if_callerid_is_present_in_calllogs_table' AIRFLOW_CTX_EXECUTION_DATE='2024-12-17T10:52:04.719125+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-17T10:52:04.719125+00:00'
[2024-12-17T10:54:11.201+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-17T10:54:11.206+0000] {base.py:84} INFO - Retrieving connection 'aicallagent'
[2024-12-17T10:54:11.210+0000] {sql.py:544} INFO - Running statement: SELECT * FROM call_logs WHERE caller_id = 'demo-scheduled-from-ui-global6a0830d8-e284-4369-9834-645e3dfae05e' limit 1;, parameters: None
[2024-12-17T10:54:11.212+0000] {sql.py:553} INFO - Rows affected: 1
[2024-12-17T10:54:11.212+0000] {python.py:240} INFO - Done. Returned value was: answered
[2024-12-17T10:54:11.222+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-17T10:54:11.222+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=call_9863800249_2024-12-17T10-52-04-719125Z, task_id=check_if_callerid_is_present_in_calllogs_table, run_id=scheduled__2024-12-17T10:52:04.719125+00:00, execution_date=20241217T105204, start_date=20241217T105411, end_date=20241217T105411
[2024-12-17T10:54:11.236+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-17T10:54:11.248+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-17T10:54:11.249+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
